rust/sdk-processor/src/main.rs

```diff
@@ -0,0 +1,28 @@
+use anyhow::Result;
+use aptos_indexer_processor_sdk_server_framework::ServerArgs;
+use clap::Parser;
+use sdk_processor::config::indexer_processor_config::IndexerProcessorConfig;
+
+#[cfg(unix)]
+#[global_allocator]
+static ALLOC: jemallocator::Jemalloc = jemallocator::Jemalloc;
+
+const RUNTIME_WORKER_MULTIPLIER: usize = 2;
+
+fn main() -> Result<()> {
+    let num_cpus = num_cpus::get();
+    let worker_threads = (num_cpus * RUNTIME_WORKER_MULTIPLIER).max(16);
```

Comment:
I understand this is copied from the current code, but I don't see any reason to do RUNTIME_WORKER_MULTIPLIER, or max(16) here.

Reply:
What do you suggest?

Reply:
I think it's ok to use just `num_cpus`.